test/src/mocha-utils.ts

```diff
@@ -389,30 +431,34 @@ export const createTimeout = <T>(
   });
 };
 
-let browserCleanups: Array<() => Promise<void>> = [];
+const browserCleanupsAfterAll: Array<() => Promise<void>> = [];
```

Comment:
Some test need to run the Cleanup after all the test so they don't spawn browser for each test;

Reply:
hm, I am not following, could you be more specific about which tests need it?

Reply:
`ignoreHTTPSErrors` and `OOPIF` Test set-up their own browsers once.
If we don't split them we get error that the browser has not been closed after each test.